db/schema.rb

```diff
@@ -29,6 +29,8 @@
     t.string   "transaction_status"
     t.datetime "created_at",         null: false
     t.datetime "updated_at",         null: false
+    t.string   "metadata"
+    t.string   "metadata_vi"
```

Comment:
In order to work with the `attr_encrypted` gem, these need to have the prefix `encrypted_`.

Reply:
This also needs to be `iv`, not `vi`